frontend/src/app/homepage.cljs

```diff
@@ -9,14 +9,14 @@
              [current-path
               remove-trailing-slash]]))
 
+(defn current-hash []
+  (. (. js/document -location) -hash))
+
 (def input-values (r/atom nil))
 (def input-errors (r/atom []))
-(def current-hash-atom (r/atom nil))
-(def backend-stats (r/atom nil))
+(def current-hash-atom (r/atom (or (str/blank? (current-hash)) "#copr" (current-hash))))(def backend-stats (r/atom nil))
```

Comment:
Somewhere a newline got lost, the `(def backend-stats (r/atom nil))` is a separate expression.

Instead of `(def current-hash-atom (r/atom (or (str/blank? (current-hash)) "#copr" (current-hash))))` it would make more sense to do `(def current-hash-atom (r/atom (or (current-hash) "#copr")))`

But otherwise, it seems to work. Thank you very much :-)

Reply:
not sure how i managed that and didnt notice 😅 It should be fixed now!